docs/rules/operator-linebreak.md

```diff
@@ -175,6 +175,26 @@ if (someCondition || otherCondition) {
 answer = everything ? 42 : foo;
 ```
 
+#### `"ignore"` override
```

Comment:
Can you put this content under "Fine grained control"? Since it only applies to overrides, it would be good to keep that information together.